CHANGELOG.md

```diff
@@ -1,7 +1,6 @@
 ## 3.64.0 (Unreleased)
 
 FEATURES:
-* **New Resource:** `google_tags_tag_binding` (beta) ([#8840](https://github.com/hashicorp/terraform-provider-google/pull/8840))
```

Comment:
This was added as beta-only but was also moved to GA this release (see the IMPROVEMENTS section below). The same is true for all the `google_tags_tag_key_iam_*` resources. I'm not sure how you want to handle that - maybe keep all of them as new features and not mention they were technically promoted?